src/it/java/teammates/it/storage/sqlapi/UsersDbIT.java

```diff
@@ -276,4 +286,301 @@ public void testGetStudentsByGoogleId()
+    }
+
+    @Test
+    public void testSqlInjectionInGetAllStudentsForEmail() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getAllStudentsForEmail email field");
+        assertEquals(usersDb.getAllStudentsForEmail(injection).size(), 0);
+    }
+
+    @Test
+    public void testSqlInjectionInGetInstructorsForGoogleId() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getInstructorsForGoogleId googleId field");
+        assertEquals(usersDb.getInstructorsForGoogleId(injection).size(), 0);
+    }
+
+    @Test
+    public void testSqlInjectionInGetStudentsForSection() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getStudentsForSection sectionName field");
+        assertEquals(usersDb.getStudentsForSection(injection, student.getCourseId()).size(), 0);
+
+        ______TS("SQL Injection test in getStudentsForSection courseId field");
+        assertEquals(usersDb.getStudentsForSection(student.getSectionName(), injection).size(), 0);
+    }
+
+    @Test
+    public void testSqlInjectionInGetStudentsForTeam() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getStudentsForTeam teamName field");
+        assertEquals(usersDb.getStudentsForTeam(injection, student.getCourseId()).size(), 0);
+
+        ______TS("SQL Injection test in getStudentsForTeam courseId field");
+        assertEquals(usersDb.getStudentsForTeam(student.getTeamName(), injection).size(), 0);
+    }
+
+    @Test
+    public void testSqlInjectionInGetStudentCountForTeam() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getStudentCountForTeam teamName field");
+        assertEquals(usersDb.getStudentCountForTeam(injection, student.getCourseId()), 0);
+
+        ______TS("SQL Injection test in getStudentCountForTeam courseId field");
+        assertEquals(usersDb.getStudentCountForTeam(student.getTeamName(), injection), 0);
+    }
+
+    @Test
+    public void testSqlInjectionInGetSection() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getSection courseId field");
+        assertNull(usersDb.getSection(injection, section.getName()));
+
+        ______TS("SQL Injection test in getSection sectionName field");
+        assertNull(usersDb.getSection(course.getId(), injection));
+    }
+
+    @Test
+    public void testSqlInjectionInGetTeam() throws Exception {
+        String injection = "test' OR 1 = 1; --";
+        ______TS("SQL Injection test in getTeam courseId field");
+        assertNull(usersDb.getSection(injection, team.getName()));
+
+        ______TS("SQL Injection test in getTeam teamName field");
+        assertNull(usersDb.getSection(course.getId(), injection));
+    }
```

Comment:
I think you forgot to change the method when copy-pasting. This doesn't test `getTeam`.

Reply:
good catch, changed!